db/models/brewType.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const db = require('APP/db');
+
+const BrewType = db.define('brews', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+
+    }
+  },
+  flavor: {
+    type: Sequelize.ARRAY(Sequelize.TEXT),
```

Comment:
keeping on both to search product and general brewTypes... recommendations on specific brands and general types.